articles/iot-hub/iot-hub-mqtt-support.md

```diff
@@ -72,9 +72,11 @@ In order to ensure a client/IoT Hub connection stays alive, both the service and
 |Node.js     |   180 seconds      |     No    |
 |Java     |    230 seconds     |     [Yes](https://github.com/Azure/azure-iot-sdk-java/blob/main/device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ClientOptions.java#L64)    |
 |C     | 240 seconds |  [Yes](https://github.com/Azure/azure-iot-sdk-c/blob/master/doc/Iothub_sdk_options.md#mqtt-transport)   |
-|C#     | 300 seconds |  [Yes](https://github.com/Azure/azure-iot-sdk-csharp/blob/main/iothub/device/src/Transport/Mqtt/MqttTransportSettings.cs#L89)   |
+|C#     | 300 seconds* |  [Yes](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.devices.client.transport.mqtt.mqtttransportsettings.keepaliveinseconds?view=azure-dotnet)   |
```

Comment:
One small change to align to our standards for linking within docs.microsoft.com:

```suggestion
|C#     | 300 seconds* |  [Yes](/dotnet/api/microsoft.azure.devices.client.transport.mqtt.mqtttransportsettings.keepaliveinseconds)   |
```

Reply:
Done.